src/gui/properties/peerlistwidget.cpp

```diff
@@ -420,7 +420,8 @@ void PeerListWidget::loadPeers(const BitTorrent::Torrent *torrent)
         for (auto i = m_peerItems.cbegin(); i != m_peerItems.cend(); ++i)
             existingPeers.insert(i.key());
 
-        const bool hideZeroValues = Preferences::instance()->getHideZeroValues();
+        const Preferences *pref = Preferences::instance();
+        const bool hideZeroValues = (pref->getHideZeroValues() && (pref->getHideZeroComboValues() == 0));
```

Comment:
That would be nice to refactor it further in order to have single accessor using `enum` type.

Reply:
> That would be nice to refactor it further in order to have single accessor using `enum` type.

Unfortunately i don't know what this means.

Reply:
> > That would be nice to refactor it further in order to have single accessor using `enum` type.
> 
> Unfortunately i don't know what this means.

This is more of a note for myself...